sipa/static/js/dormitory.js

```diff
@@ -0,0 +1,6 @@
+$("#dorm-select").change(function(){
+    selected = $("select#dorm-select").find(":selected").prop("value");
+    $(".dynamic-content").hide();
+    $("#dynamic-" + selected).show();
+});
+$("#dorm-select").on("load", function() {}).triggerHandler("change")
```

Comment:
Please adhere to your own `.editorconfig` changes (#466) and add a trailing newline :)